src/test/java/com/github/cameltooling/lsp/internal/CamelLanguageServerTest.java

```diff
@@ -122,12 +126,20 @@ public void testDONTProvideCompletionForNotCamelnamespace() throws Exception {
 	
 	@Test
 	public void testDONTProvideCompletionWhenNotAfterURIEqualQuote() throws Exception {
+		final TestLogAppender appender = new TestLogAppender();
+		final Logger logger = Logger.getRootLogger();
+        logger.addAppender(appender);
```

Comment:
indentation issue

Reply:
thanks and sorry I should have detected it myself :(